src/core/api/mcda.ts

```diff
@@ -0,0 +1,8 @@
+import { apiClient } from '~core/apiClientInstance';
+import type { AxisDTO } from '~features/bivariate_manager/types';
```

Comment:
backend-related types (DTO's) for this endpoint should be in this file

Reply:
It's not the only endpoint this type is currently used in (`/bivariate_matrix` also uses it still), so not sure I should move it out there yet.

Reply:
we do not import from features/ in core/, also we have core/bivariate/ so maybe it's a good place to collect bivariate types used across app

Reply:
Fixed